### DISCUSSION TITLE
No receiving notifications despite correct notification settings

### DISCUSSION THREAD

**Discussion by the-jasonl** [community/community#29694] [Other Feature Feedback, Questions, & Ideas]
I have been invited to an organization and have added am watching the repos, but I am not receiving any emails for any activity. My notification settings are set to receive emails. Is something not working correctly or could it be related to the organization setting (which I cannot directly view)?

**Comment by queenofcorgis** [comment]
Hi there - @the-jasonl 👋🏼, 

To get started:
* Are you receiving other email notifications from GitHub outside this organization?
* Is your GitHub email verified and set to be the primary address?
* (I know this is obvious, but I have to ask) Have you checked your spam/junk folder? 
* Are you on a work computer / any firewall issues that could be at play?

**Reply by the-jasonl** [reply]
- I have gotten an email notification for your post.
- My email is verified. The notification email is not the primary email, I have set a default notification email.
- Yes, I have checked the spam folder
- I am on a private computer and there shouldn't be any firewall issues at play

**Reply by queenofcorgis** [reply]
Thanks for taking the time to answer my questions 😄  I would recommend checking with one of the organization owners / admins to see what the notifications set up is as the next step.
